src/navigation.tpl

```diff
@@ -19,10 +19,9 @@
         </li>
         <li class="section"><a class="header" href="models.html">Models</a>
             <ul class="sub">
-                <li><a href="models-adding.html">Adding</a></li>
-                <li><a href="models-managing.html">Managing</a></li>
-                <li><a href="models-configuring.html">Additional configuration</a></li>
+                <li><a href="models-config.html">Configuring</a></li>
                 <li><a href="models-upgrade.html">Upgrading</a></li>
+                <li><a href="models-ha.html">Juju HA</a></li>
```

Comment:
Yeah, you're right. It is only HA for the model in an abstract, architectural sense.